### DISCUSSION TITLE
macOS app for LinkAce

### DISCUSSION THREAD

**Discussion by ediathome** [Kovah/LinkAce#580] [General]
Hey everyone,
first of all thank you for this cool piece of software. In case anybody is interested, I started building a macOS desktop app for use with LinkAce. You can view and download the source code here:

[https://github.com/ediathome/LinkActor](https://github.com/ediathome/LinkActor)

This is currently totally unreliable, undocumented beta software, but the basics work 🤓

Cheers

Martin

**Comment by Kovah** [comment]
Noooo wayy!!! That looks great! 🤩
Let me know if you need help from the LinkAce side.

**Reply by ediathome** [reply]
Will do - suggestions for a better name would be really welcome : )

**Reply by ediathome** [reply]
Hey Kovah,
working a on my macOS app today and came up with an API question: is it possible to load **all links in one json dataset** instead of getting paginated results?

Cheers
Martin

**Reply by Kovah** [reply]
Hi Martin, you can control the maximum number of links in the API via the `per_page` parameter. Set it to `-1` you will get all links (https://github.com/Kovah/LinkAce/blob/main/app/Helper/functions.php#L100). Caution, this might take quite a while if you have a lot links.
